src/Components/ServiceSelectionScene/ServiceSelectionScene.tsx

```diff
@@ -292,7 +310,10 @@ export class ServiceSelectionScene extends SceneObjectBase<ServiceSelectionScene
     return `{${SERVICE_NAME}=\`${service}\`}${levelFilter}`;
   }
 
-  private getMetricExpression(service: string) {
+  private getMetricExpression(service: string, serviceName: CustomConstantVariable) {
+    if (serviceName.state.value === AGGREGATED_SERVICE_NAME) {
+      return `sum by (${LEVEL_VARIABLE_VALUE}) (sum_over_time({${AGGREGATED_SERVICE_NAME}=\`${service}\`} | logfmt | drop __error__ | unwrap count [$__auto]))`;
+    }
     return `sum by (${LEVEL_VARIABLE_VALUE}) (count_over_time({${SERVICE_NAME_EXPR}=\`${service}\`} | drop __error__ [$__auto]))`;
```

Comment:
@svennergr @matyax does anyone know why we're dropping errors if we're not using a parser? 😕